tecken/symbolicate/views.py

```diff
@@ -715,6 +716,17 @@ def load_symbol(self, filename, debug_id):
         t0 = time.time()
         stream = self.get_download_symbol_stream(filename, debug_id)
 
+        # We are only interested in lines that match this regex.
+        # The 'm' is special. It's an extra *optional* prefix which,
+        # we currently omit.
+        # Origin: https://bugzilla.mozilla.org/show_bug.cgi?id=1470092
```

Comment:
I would link the upstream Breakpad issue or changeset here instead:
https://bugs.chromium.org/p/google-breakpad/issues/detail?id=751
https://chromium.googlesource.com/breakpad/breakpad/+/b1226959a25b6a5311801d6f204b088c706e7c25